### PR TITLE
Add theme to list of packages and ensure all are built on init

### PR DESCRIPTION
**Pull request by knolleary**
This adds `flowforge-nr-theme` to the list of packages that get checked out.

It also adds `flowforge` to the list of packages that get built as part of `init`.

### CODE REVIEW

File: lib/commands/init.js [hunk 1]
```diff
@@ -44,19 +44,23 @@ async function run(options) {
...
-            { cwd: path.join(options.packageDir, 'forge-ui-components') }
-        )
-        log(`${chalk.greenBright('+')} flowforge/forge-ui-components`)
-    } catch(err) {
-        log(`${chalk.greenBright('-')} flowforge/forge-ui-components`)
-        error('Failed to build forge-ui-components')
-        error(err.stderr)
-        error('Aborting')
-        return
+    const buildPackages = ['forge-ui-components','flowforge']
```
Comment (ZJvandeWeg) [inline lib/commands/init.js:47]: I wonder if build should be split out of `init` so one can invoke it after pulling new data? `init` itself invokes a subtask build then?

Mostly trying to pull `build` up to this repo as I think just building doesn't fit the `init` verb as command.

File: lib/commands/init.js [hunk 2]
```diff
@@ -44,19 +44,23 @@ async function run(options) {
...
+                `${NPM} run build`,
+                { cwd: path.join(options.packageDir, package) }
+            )
+            log(`${chalk.greenBright('+')} flowforge/${package}`)
+        } catch(err) {
+            log(`${chalk.greenBright('-')} flowforge/${package}`)
+            error(`Failed to build ${package}`)
+            error(err.stderr)
+            error('Aborting')
+            return
```
Comment (ZJvandeWeg) [inline lib/commands/init.js:62]: JS noob again, but why `return` here? And not just letting the function exit as is, given there's no more statements in it?

File: lib/commands/init.js (hunk 1 above)
Comment (knolleary) [reply lib/commands/init.js:47]: > I wonder if build should be split out

Yes - I considered that, but decided this was a good enough change for now.

When in 'development' mode, developers will typically be running `npm run serve` in the `flowforge` repo which automatically rebuilds the code - but you need to have done a one-off `npm run build` first. This is why it felt appropriate to be in the `init` task as a one-off thing.

It really depends on how we expect devs to work. I'm wary of hoisting lots of tasks out of individual repos up into this dev repo, but I'm sure there will be some more common things that make sense to break out.

File: lib/commands/init.js [hunk 3]
```diff
@@ -44,19 +44,23 @@ async function run(options) {
...
+        const package = buildPackages[i]
+        try {
+            await runner(
+                package,
+                `${NPM} run build`,
+                { cwd: path.join(options.packageDir, package) }
+            )
+            log(`${chalk.greenBright('+')} flowforge/${package}`)
+        } catch(err) {
+            log(`${chalk.greenBright('-')} flowforge/${package}`)
```
Comment (knolleary) [inline lib/commands/init.js:58]:
```suggestion
            log(`${chalk.redBright('-')} flowforge/${package}`)
```

[the remark this answers is not in the document]
Comment (knolleary) [reply lib/commands/init.js:58]: Yes - should be red. Was a copy/paste mistake in the original code, copied over here.
```suggestion
            log(`${chalk.redBright('-')} flowforge/${package}`)
```

**Pull request merged**